packages/turbo-workspaces/src/utils.ts

```diff
@@ -205,6 +201,47 @@ function getMainStep({
   } ${action === "remove" ? "from" : "to"} ${project.name}`;
 }
 
+/**
+ * At the time of writing, bun only support simple globs (can only end in /*) for workspaces. This means we can't convert all projects
+ * from other package manager workspaces to bun workspaces, we first have to validate that the globs are compatible.
+ *
+ * NOTE: It's possible a project could work with bun workspaces, but just not in the way it is currently defined. We will
+ * not change existing globs to make a project work with bun, we will only convert projects that are already compatible.
+ *
+ * This function matches the behavior of bun's glob validation: https://github.com/oven-sh/bun/blob/main/src/install/lockfile.zig#L2889
+ */
+function isCompatibleWithBunWorkspace({
+  project,
+}: {
+  project: Project;
+}): boolean {
+  const validator = (glob: string) => {
+    if (glob.includes("*")) {
+      // no multi level globs
+      if (glob.includes("**")) {
+        return false;
+      }
+
+      // no * in the middle of a path
+      const withoutLastPathSegment = glob
+        .split(path.sep)
```

Comment:
These are glob patterns, not paths. This should actually be `/`.

Reply:
Good catch